Add the Webhooks documentation (#876)
zeit/docs: pages/docs/api/v2/api-docs-mdx/endpoints/webhooks.mdx

<Example>
  <span>Example request:</span>

<Request
  method="POST"
  url="https://api.zeit.co/v1/integrations/webhooks"
  headers={{
    'Content-Type': 'application/json'
  }}
  auth
  body={{
    name: 'My first webhook',
    url: 'https://my-webhook-handler.com/zeit'
  }}
/>

Example successful (**200**) response:

<Code lang="json">{`{
  "id": "hook_XCG7t7AIHuO2SBA8667zNUiM",
  "url": "https://my-webhook-handler.com/zeit",
  "name": "My first webhook",
  "ownerId": "kr1PsOIzqEL5Xg6M4VZcZosf",
  "createdAt": 1558531915505,
  "events": []
}`}</Code>
</Example>

### Delete a Webhook

<Endpoint method="POST" url="/v1/integrations/webhooks/:id" />

Allows to create a new webhook subscription with the id of the webhooks.

### Webhook events

The webhook URL will receive an HTTP POST request with JSON payload per each event. All the events has a format like this:

<Code lang="json">{`{
  "type": <event-type>,
  "createdAt": <javascript-timestamp>,
  "payload": <payload for the event>,
  "teamId": <teamId>,
  "userId": <userId>
}`}</Code>

### Event Payloads

Here's a list of supported event types and their payload.

> #### deployment

Reviewer: ```suggestion
#### Deployment
```

| Key                 | <HelpLink href="#api-basics/types" hasIcon>Type</HelpLink> | Description                                         |
| ------------------- | ---------------------------------------------------------- | --------------------------------------------------- |

> | **deploymentId**    | <HelpLink href="#api-basics/types">ID</HelpLink>           | Id of the deployment                                |

Reviewer: ```suggestion
| **deploymentId**    | <HelpLink href="#api-basics/types">ID</HelpLink>           | The ID of the deployment.                                |
```

> | **project**         | <HelpLink href="#api-basics/types">String</HelpLink>       | Project name of the deployment                      |

Reviewer: ```suggestion
| **project**         | <HelpLink href="#api-basics/types">String</HelpLink>       | The project name of the deployment.                      |
```

> | **url**             | <HelpLink href="#api-basics/types">String</HelpLink>       | URL of the deployment                               |

Reviewer: ```suggestion
| **url**             | <HelpLink href="#api-basics/types">String</HelpLink>       | The URL of the deployment.                               |
```

> | **plan**            | <HelpLink href="#api-basics/types">String</HelpLink>       | Plan for the deployment. (unlimited, free)          |

Reviewer: ```suggestion
| **plan**            | <HelpLink href="#api-basics/types">String</HelpLink>       | The plan type of the deployment. (unlimited, free)          |
```

> | **regions**         | <HelpLink href="#api-basics/types">List</HelpLink>         | An array of supported regions.                      |

Reviewer: ```suggestion
| **regions**         | <HelpLink href="#api-basics/types">List</HelpLink>         | An array of the supported regions for the deployment.                      |
```

> | **type**            | <HelpLink href="#api-basics/types">String</HelpLink>       | The deployment type. (LAMBDAS, NPM, DOCKER, STATIC) |

Reviewer: ```suggestion
```

> | **deployment.meta** | <HelpLink href="#api-basics/types">Map</HelpLink>          | A Map of deployment metadata                        |

Reviewer: ```suggestion
| **deployment.meta** | <HelpLink href="#api-basics/types">Map</HelpLink>          | A Map of deployment metadata.                        |
```
